src/utils/feeBreakdown.test.ts

```diff
@@ -129,7 +182,27 @@ describe("toFeeBreakdown() helper function", () => {
 
     const { amount } = toFeeBreakdown(input);
 
-    expect(amount.vat).toEqual(16.67);
+    expect(amount.vat).toEqual(6);
+  });
+
+  it("correctly calculates the payable fee if there are exemptions and a service charge", () => {
+    const input: PassportFeeFields = {
+      "application.fee.calculated": 100,
+      "application.fee.payable": 36,
+      "application.fee.serviceCharge": 30,
+      "application.fee.serviceCharge.VAT": 6,
+      "application.fee.reduction.alternative": false,
+      "application.fee.reduction.parishCouncil": false,
+      "application.fee.reduction.sports": false,
+      "application.fee.exemption.disability": true,
+      "application.fee.exemption.resubmission": false,
+    };
+
+    const { amount } = toFeeBreakdown(input);
+
+    expect(amount.payable).toEqual(input["application.fee.payable"]);
+    expect(amount.vat).toEqual(input["application.fee.serviceCharge.VAT"]);
+    expect(amount.exemption).toEqual(input["application.fee.calculated"]);
```

Comment:
In order for the exemption to correctly show here, I had to swap the order of key lookup here https://github.com/theopensystemslab/planx-core/pull/694/files#diff-441beea3935905070627d583371c0f78ef44ee5f5fc7fb2e9d89ecfc707f86f4R55 (no other tests impacted by this change - so think it was a legitmate oversight/inconsistency!)